python/lsst/ip/isr/ptcDataset.py

```diff
@@ -189,11 +190,12 @@ class PhotonTransferCurveDataset(IsrCalib):
     Version 1.6 adds the `rowMeanVariance` attribute.
     Version 1.7 adds the `noiseList` attribute.
     Version 1.8 adds the `ptcTurnoffSamplingError` attribute.
+    Version 1.9 standardizes PTC noise units to electron.
     """
 
     _OBSTYPE = 'PTC'
     _SCHEMA = 'Gen3 Photon Transfer Curve'
-    _VERSION = 1.8
+    _VERSION = 1.9
```

Comment:
Look for the version checks on ptc load below and that is the place for the unit conversion.